src/utilities/process-readme.js

```diff
@@ -15,5 +15,8 @@ module.exports = function processREADME(body) {
     .replace(/<h2[^>]*>/g, '## ')
     .replace(/<\/h2>/g, '')
     // Drop any comments
-    .replace(/<!--[\s\S]*?-->/g, '');
+    .replace(/<!--[\s\S]*?-->/g, '')
+    // Replace code
+    .replace(/```console/g, '```bash')
```

Comment:
This new 2 lines save a lot of Prism warnings.

Reply:
do you think its worth it replacing content on the fly? i would suggest find replace ```console 
 -> ```bash, or is this issue coming from external readme files we fetch?

Reply:
I kinda have the feeling that we should open a PR in projects using `console` and change it instead of this because it is not a valid language option in markdown regarding highlighting.

Reply:
@EugeneHlushko Yes, it is from external readmes.
@jeremenichelli Yep :D, will revert

Reply:
We can keep this, until we replace it in all READMEs.

Reply:
I already talked to @shellscape, console is better because GitHub support is richer than `bash`.
This is easily fixed with a plugin, just registering it using Prism:
`Prism.languages[`console`] = Prism.languages.extend('sh', {})`